packages/web3-eth2-core/package.json

```diff
@@ -28,6 +28,6 @@
         "@types/node": "^14.14.22",
         "jest": "^26.5.3",
         "ts-jest": "^26.4.4",
-        "typescript": "^3.9.7"
+        "typescript": "^4.9.3"
```

Comment:
Was there a specific reason to bump this to `4.9.3` instead of matching the other packages with `v4.1`?

Reply:
Missed this thanks